.gitmodules

```diff
@@ -31,7 +31,7 @@ url = https://github.com/NGEET/fates
 fxtag = sci.1.80.11_api.37.0.0
 fxrequired = AlwaysRequired
 # Standard Fork to compare to with "git fleximod test" to ensure personal forks aren't committed
-fxDONOTUSEurl = https://github.com/NCAR/fates-release
+fxDONOTUSEurl = https://github.com/NGEET/fates
```

Comment:
@ekluzek Could you sync https://github.com/NCAR/fates-release with the upstream repo? That would then allow us to have `url` and `fxDONOTUSEurl` match, allowing the `fleximod_test` workflow to pass.

It would probably be good for all of us CTSM maintainers to have maintain permissions there too, for ease of keeping it up-to-date.

Reply:
@ekluzek likes this change, as the NCAR/FATES-release repo is not being maintained.